### ISSUE TITLE
Labels should be editable AFTER payment

### ISSUE THREAD

**Issue by dipunm** [MutinyWallet/mutiny-web#353]
Labels should be editable AFTER payment.

It is very common for people to use a new technology haphazardly and once you are happy with it, you will go back to organise things.

Labelling is not the first thing you learn, and sometimes in the moment of paying you may not even have time to think about that, so being able to add and modify labels after the fact is going to be very useful.

**Comment by futurepaul** [comment]
Absolutely, we plan to make the labeling features a lot more robust and useful
